### ISSUE TITLE
container scanning has undefined items in message

### ISSUE THREAD

**Issue by alfredodeza** [anchore/scan-action#65]
This is happening because the location of the information was not updated in `v2`:

```json
"text": "The container image contains software with a vulnerability: (undefined type=undefined)"
```

There is no `v.package` or `v.package_type`,  so it should get updated to:

```diff 
-  return prefix + "(" + v.package + " type=" + v.package_type + ")";
+  return prefix + "(" + v.artifact.name + " type=" + v.artifact.type + ")";
```

**Comment by zhill** [comment]
This should be closed by #66 , correct?

**Comment by alfredodeza** [comment]
Good catch!
